release-notes/opensearch-release-notes-1.0.0-rc1.md

```diff
@@ -0,0 +1,324 @@
+* move issue templates to ISSUE_TEMPLATE ([#1166](https://github.com/opensearch-project/security/pull/1166))
+* Rename kibana substrings with OpenSearchDashboards in class name, method name and comments ([#1160](https://github.com/opensearch-project/security/pull/1160))
+* Rename 'Open Distro' to follow open search naming convention ([#1149](https://github.com/opensearch-project/security/pull/1149))
+* Build plugin on top of 1.x branch of OpenSearch core ([#1174](https://github.com/opensearch-project/security/pull/1174))
+* Add build.version_qualifier and make security plugin compatible with OpenSearch 1.0.0-rc1 ([#1179](https://github.com/opensearch-project/security/pull/1179))
+* Update anchor link for documentation and apply opensearch-security naming convention in PR template ([#1180](https://github.com/opensearch-project/security/pull/1180))
+* Force the version of json-path 2.4.0 ([#1175](https://github.com/opensearch-project/security/pull/1175))
+* Bump version to rc1, create release notes and fix the url used in release notes drafter ([#1186](https://github.com/opensearch-project/security/pull/1186))
+* Rename settings constant value and related testing yml files for migration to Opensearch ([#1184](https://github.com/opensearch-project/security/pull/1184))
+* Remove prefix "OPENDISTRO_" for identifier for settings ([#1185](https://github.com/opensearch-project/security/pull/1185))
+* Rename documents and demo for settings ([#1188](https://github.com/opensearch-project/security/pull/1188))
+* Add fallback for opendistro_security_config.ssl_dual_mode_enabled ([#1190](https://github.com/opensearch-project/security/pull/1190))
+* Change security plugin REST API to support both opensearch and opendistro routes ([#1172](https://github.com/opensearch-project/security/pull/1172))
+* Fix CODEOWNERS file ([#1193](https://github.com/opensearch-project/security/pull/1193))
+* Dashboards rename related changes ([#1192](https://github.com/opensearch-project/security/pull/1192))
+
+
+### Opensearch Security Dashboards Plugin
+* move issue templates to ISSUE_TEMPLATE [#758](https://github.com/opensearch-project/security-dashboards-plugin/pull/758)
+* Update issue template with multiple labels [#756](https://github.com/opensearch-project/security-dashboards-plugin/pull/756)
+* Naming convention change in repo [#764](https://github.com/opensearch-project/security-dashboards-plugin/pull/764)
+* Point document links to opensearch.org [#763](https://github.com/opensearch-project/security-dashboards-plugin/pull/763)
+* Create release notes for rc-1 and Bump version to 1.0.0.0-rc1 [#771](https://github.com/opensearch-project/security-dashboards-plugin/pull/771)
+* Point OpenSearch Dashboards to to new back end APIs [#770](https://github.com/opensearch-project/security-dashboards-plugin/pull/770)
+* Change the titles of release notes [#772](https://github.com/opensearch-project/security-dashboards-plugin/pull/772)
```

Comment:
PR info not enclosed in (), not following the same standard as others.

Reply:
Done